xbmc/interfaces/legacy/Control.cpp

```diff
@@ -1036,6 +1036,11 @@ namespace XBMCAddon
       align(_alignment)
 
     {
+      dwPosX = x;
+      dwPosY = y;
+      dwWidth = width;
+      dwHeight = height;
+
```

Comment:
Should these just be done as the default constructor arguments?

```
    ControlFadeLabel::ControlFadeLabel(long x, long y, long width, long height,
                                       const char* font, const char* _textColor,
                                       long _alignment) :
      strFont("font13"), textColor(0xffffffff), align(_alignment),
      dwPosX(x),
      dwPosY(y),
      dwWidth(width),
      dwHeight(height)
    {
```

Reply:
I thought so too, but they are variables from the parent class, if i try do that compiler says:
`"dwPosX" is not a nonstatic data member or base class of class "XBMCAddon::xbmcgui::ControlEdit"`

Reply:
When I initialize parent members, I group the stanza under the heading "Initialize Control" to both inform the reader where the variables are coming from, and also in the future if this happens with multiple parents, you can verify that the most basic parent initialization comes before the least basic.